ansible/roles/stack-sunbird/templates/dhiti-service.env

```diff
@@ -0,0 +1,85 @@
+NODE_ENV={{dhiti_node_env}}
+APIDOC_URL={{dhiti_apidoc_url}}
+APIDOC_PATH={{dhiti_apidoc_path}}
+
+#Cloud storage
+CLOUD_STORAGE={{dhiti_cloud_storage}}
+BUCKET_NAME={{dhiti_bucket_name}}
+
+#Druid configuration
+DRUID_HOST={{dhiti_druid_host}}
+DRUID_PORT={{dhiti_druid_port}}
+OBSERVATION_DATASOURCE_NAME={{dhiti_observation_datasource_name}}
+ASSESSMENT_DATASOURCE_NAME={{dhiti_assessment_datasource_name}}
+ENROLLMENT_DATASOURCE_NAME={{dhiti_enrollment_datasource_name}}
+TELEMETRY_DATASOURCE_NAME={{dhiti_telemetry_datasource_name}}
+EVIDENCE_DATASOURCE_NAME={{dhiti_evidence_datasource_name}}
+SURVEY_DATASOURCE_NAME={{dhiti_survey_datasource_name}}
+SURVEY_EVIDENCE_DATASOURCE_NAME={{dhiti_survey_evidence_datasource_name}}
+OBSERVATION_EVIDENCE_DATASOURCE_NAME={{dhiti_observation_evidence_datasource_name}}
+CONTENT_REPORT_THRESHOLD={{dhiti_content_report_threshold}}
+ENTITY_SCORE_REPORT_THRESHOLD={{dhiti_entity_score_report_threshold}}
+OBSERVATION_SCORE_REPORT_THRESHOLD={{dhiti_observation_score_report_threshold}}
+ASSESSMENT_SUBMISSION_REPORT_THRESHOLD={{dhiti_assessment_submission_report_threshold}}
+EVIDENCE_THRESHOLD={{dhiti_evidence_threshold}}
+
+#Auth Config
+sunbird_keycloak_auth_server_url={{dhiti_sunbird_keycloak_auth_server_url}}
+sunbird_keycloak_realm={{dhiti_sunbird_keycloak_realm}}
+sunbird_keycloak_client_id={{dhiti_sunbird_keycloak_client_id}}
+sunbird_keycloak_public={{dhiti_sunbird_keycloak_public}}
+sunbird_cache_store={{dhiti_sunbird_cache_store}}
+sunbird_cache_ttl={{dhiti_sunbird_cache_ttl}}
+
+URL_PREFIX={{dhiti_url_prefix}}
+
+#ASSESSMENT service
+ASSESSMENT_SERVICE_APPLICATION_ENDPOINT={{dhiti_assessment_service_application_endpoint}}
+ASSESSMENT_SERVICE_BASE_URL={{dhiti_assessment_service_base_url}}
+
+#KENDRA service
+KENDRA_APPLICATION_ENDPOINT={{dhiti_kendra_application_endpoint}}
+KENDRA_BASE_URL={{dhiti_kendra_base_url}}
+
+#Gotenberg configuration
+GOTENBERG_URL={{dhiti_gotenberg_url}}
+
+#Highchart configuration
+HIGHCHART_URL={{dhiti_highchart_url}}
+
+# FILE PATHS FOR PORTAL REPORTS
+USER_NEVER_LOGGED_IN={{dhiti_user_never_logged_in}}
+UNIQUE_ACTIVE_USERS={{dhiti_unique_active_users}}
+TOP_SCORE_QUIZ={{dhiti_top_score_quiz}}
+CONTENT_RATINGS={{dhiti_content_ratings}}
+MAP_DATA_RESOURCES={{dhiti_map_data_resources}}
+LOGIN_TREND={{dhiti_login_trend}}
+LOGIN_PERCENTAGE={{dhiti_login_percentage}}
+LEARNING_TOP_SCORE_QUIZ={{dhiti_learning_top_score_quiz}}
+PARTICIPATION_PERCENTAGE={{dhiti_participation_percentage}}
+LAST_UPDATED_DATE={{dhiti_last_updated_date}}
+DAILY_AVERAGE_GROWTH={{dhiti_daily_average_growth}}
+COUNT_CONTENT_RATING={{dhiti_count_content_rating}}
+AVERAGE_TIME_SPENT={{dhiti_average_time_spent}}
+AVERAGE_RATING_CONTENT={{dhiti_average_rating_content}}
+APP_COUNT={{dhiti_app_count}}
+ADOPTION_CONTENT={{dhiti_adoption_content}}
+MULTI_RESOURCE={{dhiti_multi_resource}}
+MULTI_SELECTION={{dhiti_multi_selection}}
+
+#s3_credentials
+AWS_ACCESS_KEY_ID={{dhiti_aws_access_key_id}}
```

Comment:
We don’t have AWS in our infra. How will it work?

Reply:
It is not mandatory,
If we pass cloud storage variable as "Azure"
then this can be taken off

Reply:
Ok, in that case we can keep this section. But then where will we add azure related variables like azure account name, azure account key and azure container name?

Reply:
Once they make changes at code level to make use of Azure Storage, They will provide us the Vaiables

Reply:
Alright!